R/capture-requests.R

```diff
@@ -429,3 +434,41 @@ get_redactor <- function() {
 
   return(NULL)
 }
+
+#' Check dittodb environment path
+#'
+#' This function should generally not be used, but must be exported for the
+#' query recording function to work properly
+#'
+#' @param .dittodb_env Environment object
+#' @param n Integer. Type of error to throw.
+#'
+#' @return NULL, invisibly.
+#' @export
+check_db_path <- function(.dittodb_env, n = 1L) {
+  if(is.null(.dittodb_env$db_path)) {
```

Comment:
This is super minor, but would you mind running `styler::style_pkg()` or fixing these linting errors manually?

Reply:
applied styler